tutordistro/patches/cms-env

```diff
@@ -1,2 +1,4 @@
 USE_EOX_TENANT: True
```

Comment:
Maybe we need to create a feature to add settings to these files, because we need to remove all the EOX_ settings too.

Reply:
Ok, I will check this cases